hovor/actions/local_dialogue_action_simulated.py

```diff
@@ -0,0 +1,195 @@
+import random
+import re
+import yaml
+import json
+
+from hovor.actions.local_dialogue_action import LocalDialogueAction
+from hovor.outcome_determiners.random_outcome_determiner import RandomOutcomeDeterminer
+from hovor.outcome_determiners.workspace_outcome_determiner import WorkspaceOutcomeDeterminer
+
+
+class LocalDialogueActionSimulated(LocalDialogueAction):
+    """Dialogue type of action - simulated version."""
+
+    _apply_message_grouping = False
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        self.is_external = False
+        self._utterance = "HOVOR: " + \
+            random.choice(self.config["message_variants"])
+        self._utterance = LocalDialogueActionSimulated.replace_pattern_entities(
+            self._utterance, self.context)
+        # this is only used in the deprecated method of generating a response
+        # with open("/home/jacob/Dev/plan4dial/plan4dial/local_data/gold_standard_bot/output_files/nlu.yml") as fin:
+        #     intents = yaml.safe_load(fin)
+        # self.intents = intents['nlu']
+        # This is used in the process of generating a response
+        with open("/home/jacob/Dev/plan4dial/plan4dial/local_data/gold_standard_bot/output_files/data.json") as fin:
```

Comment:
I agree it isn't great the have that hardcoded. I have spent some time looking at how to parameterize it and it is proving to be tricky, because the call stack
![image](https://user-images.githubusercontent.com/38571487/213943781-e428b2e7-4319-4f5b-b9a6-14d86a1b72e7.png)

Goes through a number of files that I don't think should be modified for simulation. 

I settled on a change in json_configuration_provider to include a copy of the data when the action is instantiated. I actually realize I don't need to even read from the file, that is actually already done bu hovor when specifying output files.